src/settings.py

```diff
@@ -206,8 +206,10 @@ def get_arg_specific_settings(args: Dict[str, Any]) -> Dict[str, Any]:
 def get_allowed_query_type_names() -> Set[str]:
     """Get the set of names of query types that the teacher is allowed to
     answer, computed based on the configuration CFG."""
-    if CFG.approach in ("interactive_learning", "unittest"):
+    if CFG.option_learner == "neural":
```

Comment:
we still need a case for `interactive_learning` here!

Reply:
oh gotcha!